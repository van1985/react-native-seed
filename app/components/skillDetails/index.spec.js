import React from 'react';
import renderer from 'react-test-renderer';
import SkillDetails from './index';

const skill = {
  Name: 'John',
  percentage: '80',
};

test('renders correctly', () => {
  const tree = renderer.create(<SkillDetails
    percentage={skill.percentage}
    skillName={skill.name}
    key={skill.name}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
