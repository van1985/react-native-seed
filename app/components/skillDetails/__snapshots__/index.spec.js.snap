// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={undefined}
>
  <View
    style={
      Object {
        "display": "flex",
        "flexDirection": "row",
        "marginTop": 20,
        "paddingLeft": 20,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "paddingTop": 8,
          "width": 80,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#ffffff",
            "fontSize": 24,
            "paddingBottom": 10,
          }
        }
      >
        80
        %
      </Text>
    </View>
    <View
      style={
        Object {
          "paddingLeft": 10,
          "width": 400,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#ffffff",
            "fontSize": 18,
            "paddingBottom": 10,
          }
        }
      />
      <View
        style={
          Array [
            Object {
              "width": 240,
            },
            Object {
              "backgroundColor": "#666666",
              "height": 6,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "width": 40,
              },
              Object {
                "backgroundColor": "#00AEEF",
                "height": 6,
              },
            ]
          }
        />
      </View>
    </View>
  </View>
</View>
`;
